apps/homescreen/manifest.webapp

```diff
@@ -21,11 +21,8 @@
   },
   "permissions": {
     "homescreen-webapps-manage": {},
-    "mobileconnection": {},
     "moz-extremely-unstable-and-will-change-webcomponents": {},
-    "open-remote-window": {},
```

Comment:
open-remote-window does actually work for privileged home-screens, the check is done in Gaia system.